src/Notifier.js

```diff
@@ -192,12 +216,22 @@ var Notifier = {
         return enabled === 'true';
     },
 
-    setToolbarHidden: function(hidden) {
+     setToolbarHidden: function(hidden, persistent) {
         this.toolbarHidden = hidden;
         dis.dispatch({
             action: "notifier_enabled",
             value: this.isEnabled()
         });
+
+        if (persistent === true)
```

Comment:
I'd just make this `if (persistent)`. Also please use `{}` even for one-line if statements.